src/node_platform.h

```diff
@@ -88,7 +88,7 @@ class PerIsolatePlatformData :
   void DecreaseHandleCount();
 
   static void FlushTasks(uv_async_t* handle);
-  static void RunForegroundTask(std::unique_ptr<v8::Task> task);
+  void RunForegroundTask(std::unique_ptr<v8::Task> task);
```

Comment:
Is there any advantage of converting a static into a non-static?

Reply:
@gireeshpunathil Yes, because this method now accesses the `isolate_` field.